utils/settings.ts

```diff
@@ -165,6 +165,18 @@ export const formatCurrencies = (settings: any): Currency[] =>
     })
     .filter(isNotNull) ?? [];
 
+export const formatCurrency = (settings: any) => {
```

Comment:
This method does not seem to do any formatting.  Should this be named `resolveCurrency` or `getCurrencyFromSettings`?

Reply:
renamed to getCurrencyFromSettings